schematics/src/ng-add/index.ts

```diff
@@ -148,6 +182,25 @@ function addModuleToImports(options: Schema): Rule {
       addModuleImportToRootModule(host, importRouter, null as any, project);
     }
 
+    if (provideEntityServiceConfig) {
```

Comment:
Why not use `addModuleImportToRootModule`?

Reply:
It's not a module, it's a provider.